doc/content/lorawan/rssi-and-snr/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "RSSI and SNR"
+section: Additional Information
+description: ""
+weight:
+---
+
+In wireless communication, a receiver needs a good signal strength and a signal-to-noise ratio to separate the original information or signal from the modulated carrier.
+
+## RSSI
+
+**RSSI (Received Signal Strength Indicator)** is a measurement that helps you to determine if the receiver is receiving enough signal (signal strength) to get a good wireless connection from the transmitter. Since LoRaWAN supports bi-directional communication, RSSI is an important measurement for both gateways and end devices. RSSI is measured in dBm and it is a negative value. An RSSI value that is closer to zero usually means a better signal strength.
```

Comment:
```suggestion
RSSI (Received Signal Strength Indicator) is a relative measurement that helps you determine if the received signal is strong enough to get a good wireless connection from the transmitter. Since LoRaWAN supports bi-directional communication, RSSI is an important measurement for both gateways and end devices. RSSI is measured in dBm and its value is a negative form. The closer the RSSI value is to zero, the received signal is stronger.
```

Reply:
also link this https://www.thethingsnetwork.org/docs/lorawan/db-dbm-dbi-dbd/#dbm-decibel-per-milliwatt on dBm